pdf-aam/index.html

```diff
@@ -191,33 +141,40 @@ <h2>General rules</h2>
 
         <section id="mapping_role_table">
           <h3>Role Mapping Tables</h3>
-          <h4 id="role-map-example"><code>example</code></h4>
+          <h4 id="role-map-example"><code>Document Logical document within a parent document</code></h4>
           <table class="data" aria-labelledby="role-map-example">
             <tbody>
               <tr>
                 <th>ARIA Specification</th>
                 <td>
-                  <a class="role-reference" href="#example"><code>example</code></a>
+                  <a class="role-reference" href="https://www.w3.org/TR/core-aam-1.2/#role-map-generic"><code>generic </code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>Analogous HTML Element</th>
+                <td>
+                  <span class="property" href="Use WAI-ARIA mapping">Role: <code>Use WAI-ARIA mapping</code></span
+                  ><br />
                 </td>
               </tr>
               <tr>
                 <th>MSAA + IAccessible2</th>
                 <td>
-                  <span class="property">Role: <code>ROLE_SYSTEM_EXAMPLE</code></span
+                  <span class="property">Role: <code>Use WAI-ARIA mapping</code></span
                   ><br />
                 </td>
               </tr>
               <tr>
                 <th><abbr title="User Interface Automation">UIA</abbr></th>
                 <td>
-                  <span class="property">Control Type: <code>Example</code></span
+                  <span class="property">Control Type: <code>Use WAI-ARIA mapping</code></span
```

Comment:
```suggestion
<div class="general">Use WAI-ARIA mapping</div>
```
You don't need the soft line breaks or `<br />` either, FWIW